docs/tutorials/csharp/basic_csharp.md

```diff
@@ -10,73 +10,67 @@ nav_order: 1
 
 # C# Tutorial: Basic
 
-Here is a simple tutorial for getting started with running inference on an existing ONNX model for a given input data.
-The model is typically trained using any of the well-known training frameworks and then exported into the ONNX format. 
+Learn how to get started with inference with the C# API. Run an ONNX model for a given input data. The model is typically trained using any of the well-known training frameworks and then exported into the ONNX format.
```

Comment:
I think we can simplify this section to just a single sentence: "Learn how to get started with inferencing ONNX models using the ONNX Runtime C# API"